src/main/java/fly/wild/wizards/serverside/ServerConnnector.java

```diff
@@ -190,93 +117,208 @@ public boolean configureOneWayTLS () {
-		coreServiceSSLContext.get(ClientConstants.OP_ADDR).add(STRING_MANAGEMENT_INTERFACE_ATTRIBUTE,STRING_MANAGEMENT_INTERFACE_VALUE);
-		coreServiceSSLContext.get(STRING_NAME_ATTRIBUTE).set("ssl-context");
-		coreServiceSSLContext.get("value").set(this.genServerSSLContext);
-		
-		coreServiceSecureSocketBinding.get(ClientConstants.OP).set(ClientConstants.WRITE_ATTRIBUTE_OPERATION);
-		coreServiceSecureSocketBinding.get(ClientConstants.OP_ADDR).add("core-service","management");
-		coreServiceSecureSocketBinding.get(ClientConstants.OP_ADDR).add("management-interface","http-interface");
-		
-		coreServiceSecureSocketBinding.get("name").set("secure-socket-binding");
-		coreServiceSecureSocketBinding.get("value").set("management-https");
-		
+		DefaultOperationRequestBuilder coreServiceSecureSocketBindingBuilder = new DefaultOperationRequestBuilder();
+		coreServiceSecureSocketBindingBuilder.setOperationName(Util.WRITE_ATTRIBUTE);
+		coreServiceSecureSocketBindingBuilder.addNode(Util.CORE_SERVICE,Util.MANAGEMENT);
+		coreServiceSecureSocketBindingBuilder.addNode(Util.MANAGEMENT_INTERFACE,Util.HTTP_INTERFACE);
+		coreServiceSecureSocketBindingBuilder.addProperty(Util.NAME,Util.SECURE_SOCKET_BINDING);
+		coreServiceSecureSocketBindingBuilder.addProperty(Util.VALUE,Util.MANAGEMENT_HTTPS);
+
 		try {
-			this.client = ModelControllerClient.Factory.create (InetAddress.getByName(this.ipAddress), 
-					DMR_PORT);
 			
-			client.execute(new OperationBuilder(coreServiceSSLContext).build());
-			client.execute(new OperationBuilder(coreServiceSecureSocketBinding).build());
-		} catch (IOException e) {
+			ModelControllerClient client = ModelControllerClientFactory.CUSTOM.getClient(connectionAddress, null, false, null, false, 0, null, null, null);
+			ModelNode coreService = client.execute(coreServiceSSLContextBuilder.buildRequest());
+			this.logMessage.append("Configuring coreServiceSSLContext services\n");
+			this.logMessage.append(coreService.toString()+"\n");
+			ModelNode secureSocketBinding = client.execute(coreServiceSecureSocketBindingBuilder.buildRequest());
+			this.logMessage.append("Configuring secure socket binding\n");
+			this.logMessage.append(secureSocketBinding.toString()+"\n");
+			
+		} catch (IOException | OperationFormatException e) {
 			// TODO Auto-generated catch block
 			e.printStackTrace();
 		}
 
 		
 	}
 	
+	public void reloadServer () {
+		DefaultOperationRequestBuilder reloadBuilder = new DefaultOperationRequestBuilder();
+		reloadBuilder.setOperationName(Util.RELOAD);
+		
+		ModelControllerClient client;
+		try {
+			client = ModelControllerClientFactory.CUSTOM.getClient(connectionAddress, null, false, null, false, 0, null, null, null);
+			client.execute(reloadBuilder.buildRequest());
+			this.logMessage.append("Server reloaded\n");
+		} catch (IOException | OperationFormatException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
```

Comment:
just a reminder that this TODO is here. It is better to not print the stacktrace but rather have some messages instead for a user that mention what could have gone wrong

Reply:
Thanks, i created an issue for this: https://github.com/theashiot/tlswizard/issues/6